class.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Please don't disable linters; they are there to help you adhere to a coding standard 👍